### PR TITLE
[GA Skills SDK][Virtual Assistant] Update DefaultAdapter

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts [hunk 1]
```diff
@@ -33,6 +34,7 @@ export class DefaultAdapter extends BotFrameworkAdapter {
         adapterSettings: Partial<BotFrameworkAdapterSettings>,
         telemetryMiddleware: TelemetryInitializerMiddleware,
         telemetryClient: BotTelemetryClient
+        // PENDING: Pass IChannelProvider to BotFrameworkHttpAdapter but neither exists in BotBuilder-JS
```
Comment (dfavretto) [inline templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts]: After the discussion and investigation done along with @VictorGrycuk  about `IChannelProvider` and `BotFrameworkHttpAdapter`, this line can be fixed.

Comment (VictorGrycuk) [reply templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts]: The comment was removed, the necessary property was added to `index.ts`

File: templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts [hunk 2]
```diff
@@ -67,5 +68,6 @@ export class DefaultAdapter extends BotFrameworkAdapter {
         this.use(new FeedbackMiddleware(conversationState, telemetryClient));
```
Comment (Batta32) [inline templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts]: Can we pass the feedbackOptions as [C#]
(https://github.com/microsoft/botframework-solutions/blob/10ae4d2f0fa595e9d1b1f5d5ba11f15fd3160e9a/templates/csharp/VA/VA/Adapters/DefaultAdapter.cs#L44)?

Also, this line should be above line 71

File: templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts [hunk 3]
```diff
@@ -67,5 +68,6 @@ export class DefaultAdapter extends BotFrameworkAdapter {
...
         this.use(new SetLocaleMiddleware(settings.defaultLocale || 'en-us'));
```
Comment (Batta32) [inline templates/typescript/samples/sample-assistant/src/adapters/defaultAdapter.ts]: This should be in the line 68

**Comment by Batta32** [comment]
LGTM, @dfavretto just merge it when you want.

**Pull request merged**